### PR TITLE
feat: add navbar

### LINKED ISSUE #1: add navbar
here I added responsive navbar

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added  navbar which is responsive. It containes search bar.
Screenshot (1335)

### CODE REVIEW

**Review by ShreyashMahajan — APPROVED** [review]
code looks clean and understoodable. Good to go

File: src/Components/HamburgurMenu/HamburgurMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import "./HamburgurMenu.css";
+import { Link } from "react-router-dom";
+import { HiHome } from "react-icons/hi";
+import { FaUserAlt } from "react-icons/fa";
+import { MdVideoLibrary, MdOutlinePlaylistPlay } from "react-icons/md";
+import { AiFillLike } from "react-icons/ai";
+import { IoIosInformationCircle } from "react-icons/io";
+import { RiPlayList2Fill } from "react-icons/ri";
+import { BsFillBookmarkFill } from "react-icons/bs";
+export const HamburgurMenuList = () => {
```
Comment (tushartiwari7) [inline src/Components/HamburgurMenu/HamburgurMenu.jsx:11]: I will suggest you to either use one single library for icons or import all icons in a separate file and use it from there.

**Review by tushartiwari7 — APPROVED** [review]
Have a look at the suggestions and rest code looks clean!

File: src/Components/HamburgurMenu/HamburgurMenu.jsx (hunk 1 above)
Comment (Bhakti199) [reply src/Components/HamburgurMenu/HamburgurMenu.jsx:11]: nice idea, thanks tushar.

**Pull request merged**